integration_test/src/main.rs

```diff
@@ -778,6 +786,97 @@ fn test_rescan_blockchain(cl: &Client) {
     assert_eq!(stop, Some(count - 1));
 }
 
+struct WalletParams<'a> {
+    name: &'a str,
+    disable_private_keys: Option<bool>,
+    blank: Option<bool>,
+    passphrase: Option<&'a str>,
+    avoid_reuse: Option<bool>,
+}
+
+fn build_wallet_params<'a>(
```

Comment:
This is a little bit of a silly utility method that I'd personally not have created, but I don't bother because it's test stuff :)

Reply:
Removed, didn't have very strong feelings about it :)